src/code42cli/cmds/legal_hold.py

```diff
@@ -25,6 +27,18 @@
 _MATTER_KEYS_MAP["description"] = "Description"
 _MATTER_KEYS_MAP["creator_username"] = "Creator"
 _MATTER_KEYS_MAP["creationDate"] = "Creation Date"
+_EVENT_KEYS_MAP = OrderedDict()
```

Comment:
We no longer need to use `OrderedDict` in the CLI, we no longer support any python version that requires that.
Regular `{}` syntax should suffice

Reply:
The above `_MATTER_KEYS_MAP`  could be changed as well